### DISCUSSION TITLE
Is there anyway to disable reloading while the popup bar is open and presenting?

### DISCUSSION THREAD

**Discussion by dfeler** [LeoNatan/LNPopupController#505] [General] [answered]
It appears ViewController is reloading often when presented by the LNPopuController, meaning the popup could be in an open state and the view controller refreshes and reloads as if it was never opened before.

ViewDidLoad, ViewDidAppear..etc.. Get executed again all while the user is looking at the view.

This is effecting the MapView I have in the presenting view controller, after a minute or so the map will reload and change the position the user had it in.

Is there anyway to disable reloading while the popup bar is open and presenting?

**Comment by LeoNatan** [comment, marked as answer]
It means you are presenting a new popup content controller every time you refresh. You need to manage your app state and logic properly.

Please note that the Discussions and Issues sections are not meant to teach users how to code their apps. Sorry.
